public/app/percona/integrated-alerting/types.ts

```diff
@@ -7,4 +9,14 @@ declare module 'react-table' {
     isExpanded: boolean;
     getToggleRowExpandedProps?: () => void;
   }
+
+  interface HeaderGroup {
+    className?: string;
```

Comment:
These two seem like the same thing

Reply:
Indeed, but in fact here I'm merging these with react-table's own typings, because those are a bit outdated